x-pack/test/cases_api_integration/security_and_spaces/tests/common/cases/post_case.ts

```diff
@@ -482,59 +484,80 @@ export default ({ getService }: FtrProviderContext): void => {
           );
         });
 
-        it('400s when creating a case with a missing required custom field', async () => {
+        it('creates a case with missing required custom fields and default values', async () => {
```

Comment:
This test does not test an unhappy path anymore so I think is best if we move it outside the unhappy block.